webapp/src/gitjson.tsx

```diff
@@ -0,0 +1,693 @@
+            pxt.reportException(e);
+            core.warningNotification(lf("Oops, something went wrong. Please try again."))
+        }
+    }
+
+    private async bumpAsync() {
+        const v = pxt.semver.parse(pkg.mainPkg.config.version || "0.0.0")
+        const vmajor = pxt.semver.parse(pxt.semver.stringify(v)); vmajor.major++; vmajor.minor = 0; vmajor.patch = 0;
+        const vminor = pxt.semver.parse(pxt.semver.stringify(v)); vminor.minor++; vminor.patch = 0;
+        const vpatch = pxt.semver.parse(pxt.semver.stringify(v)); vpatch.patch++;
+
+        let bumpType: string = "patch";
+        function onBumpChange(e: React.ChangeEvent<HTMLInputElement>) {
+            bumpType = e.currentTarget.name;
+            coretsx.forceRender();
+        }
+        const ok = await core.confirmAsync({
+            header: lf("Pick a release version"),
+            agreeLbl: lf("Create release"),
+            disagreeLbl: lf("Cancel"),
+            jsxd: () => <div className="grouped fields">
+                <label>{lf("Choose a release version that describes the changes you made to the code.")}
+                    <sui.Link href="https://makecode.com/extensions/versioning" icon="help circle" target="_blank" role="button" title={lf("Learn about version numbers.")} />
+                </label>
+                <div className="field">
+                    <div className="ui radio checkbox">
+                        <input type="radio" name="patch" checked={bumpType == "patch"} aria-checked={bumpType == "patch"} onChange={onBumpChange} />
+                        <label>{lf("{0}: patch (bug fixes or other non-user visible changes)", pxt.semver.stringify(vpatch))}</label>
+                    </div>
+                </div>
+                <div className="field">
+                    <div className="ui radio checkbox">
+                        <input type="radio" name="minor" checked={bumpType == "minor"} aria-checked={bumpType == "minor"} onChange={onBumpChange} />
+                        <label>{lf("{0}: minor change (added function or optional parameters)", pxt.semver.stringify(vminor))}</label>
+                    </div>
+                </div>
+                <div className="field">
+                    <div className="ui radio checkbox">
+                        <input type="radio" name="major" checked={bumpType == "major"} aria-checked={bumpType == "major"} onChange={onBumpChange} />
+                        <label>{lf("{0}: major change (renamed functions, deleted parameters or functions)", pxt.semver.stringify(vmajor))}</label>
+                    </div>
+                </div>
+            </div>
+        })
+
+        if (!ok)
+            return
+
+        let newv = vpatch;
+        if (bumpType == "major")
+            newv = vmajor;
+        else if (bumpType == "minor")
+            newv = vminor;
+        const newVer = pxt.semver.stringify(newv)
+        core.showLoading("bumpheader", lf("creating release {0}...", newVer));
```

Comment:
Capitalize

Reply:
all loading message should be lower cased; just like "loading editor..." or "loading...".